go.mod

```diff
@@ -4,6 +4,8 @@ go 1.14
 
 require (
 	github.com/gruntwork-io/terratest v0.30.0
+	github.com/homeport/dyff v1.2.2 // indirect
```

Comment:
Not a huge deal, but do we want to add these deps? It doesn't look like they're used in the code

Reply:
kubeval makes sense, but are you using dyff?

Reply:
I’ve yanked `dyff`, but have added a test with `kubeval`.